internal/store/cronjob_test.go

```diff
@@ -33,6 +33,8 @@ var (
 	SuspendTrue                      = true
 	SuspendFalse                     = false
 	StartingDeadlineSeconds300 int64 = 300
+	SuccessfulJobHistoryLimit3 int32 = 3
```

Comment:
why a value 3 being used?

Reply:
This is just a unit test we can keep any value just have to make sure use the same for verification. Used 3 as it's the default value for this field in Kubernetes if unspecified.